dbconnect_example.py

```
@@ -0,0 +1,5 @@
+from databricks.connect import DatabricksSession
+
+spark = DatabricksSession.builder.profile("mlops-dev-cli").getOrCreate()
+df = spark.read.table("samples.nyctaxi.trips")
+df.show(5)
```

Ruff

No newline at end of file

Add trailing newline

(W292)